### PR TITLE
Completed functionality on submission, will go to next page.

### PR DESCRIPTION
**Pull request by ocampd1**
Added component and service for submit as well.

### CODE REVIEW

File: chucks-pick3-web/angular-app/src/app/components/sign-up.component/sign-up.component.ts [hunk 1]
```diff
@@ -1,19 +1,47 @@
 import {Component, OnInit} from '@angular/core';
-
+import {FormValidationService} from '../../services/form-validation.service/form-validation.service';
+import {MemberService} from '../../services/member.service/member.service';
```
Comment (pouncilt) [inline chucks-pick3-web/angular-app/src/app/components/sign-up.component/sign-up.component.ts]: Lets ensure all path start with `src/app/`; so the path should be like so: `'src/app/services/member.service/member.service'`.  Please update all other paths are corrected in all TS files.

File: chucks-pick3-web/angular-app/src/app/components/sign-up.component/sign-up.component.ts [hunk 2]
```diff
@@ -1,19 +1,47 @@
...
   selector: 'app-sign-up',
   templateUrl: './sign-up.component.html',
   styleUrls: ['./sign-up.component.css']
 })
 export class SignUpComponent implements OnInit {
+  // Todo: create an errorMessage attribute so that it can be use later if and when the memberService.createAccount() return false.
+ public setErrorMessage(querySelector, errorMessage, input) {
+    document.querySelector(querySelector).innerHTML = errorMessage;
+    input.setCustomValidity(errorMessage);
+
```
Comment (pouncilt) [inline chucks-pick3-web/angular-app/src/app/components/sign-up.component/sign-up.component.ts]: Remove empty space.  Look in other files for empty spaces and remove them too.

File: chucks-pick3-web/angular-app/src/app/components/sign-up.component/sign-up.component.ts [hunk 3]
```diff
@@ -1,19 +1,47 @@
...
+   const isFormValid =  this.formValidationService.validateForm();
+   if (isFormValid === true) {
+     isFormValid.memberService.createAccount();
+   } else if (isFormValid === false) {
+     this.setErrorMessage(".invalid-feedback." + input.name, '', input);
+   }
+
+  }
+
+  // Todo: create onSubmit() method that delegates call to:
```
Comment (pouncilt) [inline chucks-pick3-web/angular-app/src/app/components/sign-up.component/sign-up.component.ts]: Remove comments and todos in all TS files.

File: chucks-pick3-web/angular-app/src/app/services/form-validation.service/form-validation.service.ts [hunk 4]
```diff
@@ -7,10 +7,14 @@ export class FormValidationService {
   constructor() {
   }
 
+/*ADD validateForm here!*/
+public validateForm() {
+return true;
```
Comment (pouncilt) [inline chucks-pick3-web/angular-app/src/app/services/form-validation.service/form-validation.service.ts]: Be sure to have proper indentions.  Check all TS files for proper indentions.

File: chucks-pick3-web/static/assets/js/bootstrapFormVal.js [hunk 5]
```diff
@@ -7,7 +7,7 @@
         var forms = document.getElementsByClassName('needs-validation');
         // Loop over them and prevent submission
         var validation = Array.prototype.filter.call(forms, function (form) {
-            form.addEventListener('submit', function (event) {
+            forms.addEventListener('submit', function (event) {
```
Comment (pouncilt) [inline chucks-pick3-web/static/assets/js/bootstrapFormVal.js]: Revert this change back to using just `form` instead of `forms`.

Comment (ocampd1) [reply chucks-pick3-web/static/assets/js/bootstrapFormVal.js]: Completed changes for form.

File: chucks-pick3-web/angular-app/src/app/services/form-validation.service/form-validation.service.ts (hunk 4 above)
Comment (ocampd1) [reply chucks-pick3-web/angular-app/src/app/services/form-validation.service/form-validation.service.ts]: Formatted text accordingly.

File: chucks-pick3-web/angular-app/src/app/components/sign-up.component/sign-up.component.ts (hunk 3 above)
Comment (ocampd1) [reply chucks-pick3-web/angular-app/src/app/components/sign-up.component/sign-up.component.ts]: Removed all comments and todos.

File: chucks-pick3-web/angular-app/src/app/components/sign-up.component/sign-up.component.ts (hunk 2 above)
Comment (ocampd1) [reply chucks-pick3-web/angular-app/src/app/components/sign-up.component/sign-up.component.ts]: Looked for empty space and formatted code

File: chucks-pick3-web/angular-app/src/app/components/sign-up.component/sign-up.component.ts (hunk 1 above)
Comment (ocampd1) [reply chucks-pick3-web/angular-app/src/app/components/sign-up.component/sign-up.component.ts]: Added src/app/ to paths.

File: chucks-pick3-web/angular-app/src/app/components/home.component/home.component.html [hunk 6]
```diff
@@ -9,7 +9,7 @@
   <div class="col-md-4 no-pad-right">
     <a [routerLink]="['/requirements']" [queryParams]="{showSubNavBar: true}">
       <div class="grediant">
-        <img src="assets/images/screens-02.png">
+        <img src="src/app/assets/images/screens-02.png">
```
Comment (pouncilt) [inline chucks-pick3-web/angular-app/src/app/components/home.component/home.component.html]: Not Sure about this change.  Does this still work?  I think it is okay to start paths to images and css files with just `assets/images` and `assets/css`;  Please revert back.

File: chucks-pick3-web/angular-app/src/app/components/home.component/home.component.html [hunk 7]
```diff
@@ -20,18 +20,19 @@
   <div class="col-md-4 middlebdr no-pad">
     <a [routerLink]="['/demo']" [queryParams]="{showSubNavBar: false}">
       <div class="grediant">
-        <img src="assets/images/screens-01.png">
+        <img src="src/app/assets/images/screens-01.png">
```
Comment (pouncilt) [inline chucks-pick3-web/angular-app/src/app/components/home.component/home.component.html]: Not Sure about this change.  Does this still work?  I think it is okay to start paths to images and css files with just `assets/images` and `assets/css`;  Please revert back.

File: chucks-pick3-web/angular-app/src/app/components/home.component/home.component.html [hunk 8]
```diff
@@ -20,18 +20,19 @@
...
+          tristique semper. Nullam lacinia
+          tincidunt lectus vel volutpat.</p>
       </div>
     </a>
   </div>
   <div class="col-md-4 no-pad-left">
     <a [routerLink]="['/sign-up']" [queryParams]="{showSubNavBar: false}">
       <div class="grediant">
-        <img src="/assets/images/screens_Artboard 1-03.png" class="signup" alt="Sign-Up" />
+        <img src="src/app/assets/images/screens_Artboard 1-03.png" class="signup" alt="Sign-Up"/>
```
Comment (pouncilt) [inline chucks-pick3-web/angular-app/src/app/components/home.component/home.component.html]: Not Sure about this change.  Does this still work?  I think it is okay to start paths to images and css files with just `assets/images` and `assets/css`;  Please revert back.

File: chucks-pick3-web/angular-app/src/app/components/info-generate.component/info-generate.component.html [hunk 9]
```diff
@@ -2,7 +2,7 @@
   <h1><b>How to Generate Picks:</b></h1>
 </div>
 <div class="how-to-generate">
-  <img src="/assets/images/Info page -12.png" alt="Generate" style="width:700px" align="center">
+  <img src="src/app/assets/images/Info page -12.png" alt="Generate" style="width:700px" align="center">
```
Comment (pouncilt) [inline chucks-pick3-web/angular-app/src/app/components/info-generate.component/info-generate.component.html]: Not Sure about this change.  Does this still work?  I think it is okay to start paths to images and css files with just `assets/images` and `assets/css`;  Please revert back.

File: chucks-pick3-web/angular-app/src/app/components/info-generate.component/info-generate.component.html [hunk 10]
```diff
@@ -2,7 +2,7 @@
...
 
-  <img src="/assets/images/Info page -13.png" alt="Generate" style="width:700px" align="center">
+  <img src="src/app/assets/images/Info page -13.png" alt="Generate" style="width:700px" align="center">
```
Comment (pouncilt) [inline chucks-pick3-web/angular-app/src/app/components/info-generate.component/info-generate.component.html]: Not Sure about this change.  Does this still work?  I think it is okay to start paths to images and css files with just `assets/images` and `assets/css`;  Please revert back.

File: chucks-pick3-web/angular-app/src/app/components/info-interface.component/info-interface.component.html [hunk 11]
```diff
@@ -1,10 +1,10 @@
 <div class="how-to-header" id="interface">
   <h1><b>Interface:</b></h1>
-  <img src="/assets/images/Info page -04.png" alt="Interface" style="width:700px" align="middle">
+  <img src="src/app/assets/images/Info page -04.png" alt="Interface" style="width:700px" align="middle">
```
Comment (pouncilt) [inline chucks-pick3-web/angular-app/src/app/components/info-interface.component/info-interface.component.html]: Not Sure about this change.  Does this still work?  I think it is okay to start paths to images and css files with just `assets/images` and `assets/css`;  Please revert back.

File: chucks-pick3-web/angular-app/src/app/components/info-interface.component/info-interface.component.html [hunk 12]
```diff
@@ -1,10 +1,10 @@
...
 </div>
 <br>
 <br>
 <div class="interface">
-  <img src="/assets/images/Info page -15.png" alt="Interface" style="width:700px" align="middle">
+  <img src="src/app/assets/images/Info page -15.png" alt="Interface" style="width:700px" align="middle">
```
Comment (pouncilt) [inline chucks-pick3-web/angular-app/src/app/components/info-interface.component/info-interface.component.html]: Not Sure about this change.  Does this still work?  I think it is okay to start paths to images and css files with just `assets/images` and `assets/css`;  Please revert back.

File: chucks-pick3-web/angular-app/src/app/components/info-requirements.component/info-requirements.component.html [hunk 13]
```diff
@@ -1,6 +1,6 @@
 <div class="how-to-header" id="requirements">
   <h1><b>Requirements:</b></h1>
-    </div>
-    <div class="what-you-need">
-  <img src="/assets/images/Info page -11.png" alt="IPhone">
+</div>
+<div class="what-you-need">
+  <img src="src/app/assets/images/Info page -11.png" alt="IPhone">
```
Comment (pouncilt) [inline chucks-pick3-web/angular-app/src/app/components/info-requirements.component/info-requirements.component.html]: Not Sure about this change.  Does this still work?  I think it is okay to start paths to images and css files with just `assets/images` and `assets/css`;  Please revert back.

File: chucks-pick3-web/angular-app/src/app/components/info-view.component/info-view.component.html [hunk 14]
```diff
@@ -1,6 +1,6 @@
 <div class="how-to-header" id="generated-picks">
   <h1><b>How to View Generated Picks:</b></h1>
 </div>
-  <div class="how-to-view">
-  <img src="/assets/images/Info page -14.png" alt="ViewGenerated" style="width:700px" align="center">
+<div class="how-to-view">
+  <img src="src/app/assets/images/Info page -14.png" alt="ViewGenerated" style="width:700px" align="center">
```
Comment (pouncilt) [inline chucks-pick3-web/angular-app/src/app/components/info-view.component/info-view.component.html]: Not Sure about this change.  Does this still work?  I think it is okay to start paths to images and css files with just `assets/images` and `assets/css`;  Please revert back.

File: chucks-pick3-web/angular-app/src/app/components/info.component/info.component.html [hunk 15]
```diff
@@ -2,5 +2,5 @@
   <h1><b>Requirements:</b></h1>
 </div>
 <div class="what-you-need">
-  <img src="/assets/images/Info page -11.png" alt="IPhone">
+  <img src="src/app/assets/images/Info page -11.png" alt="IPhone">
```
Comment (pouncilt) [inline chucks-pick3-web/angular-app/src/app/components/info.component/info.component.html]: Not Sure about this change.  Does this still work?  I think it is okay to start paths to images and css files with just `assets/images` and `assets/css`;  Please revert back.

File: chucks-pick3-web/angular-app/src/app/shared/components/app-header.component/app-header.component.html [hunk 16]
```diff
@@ -3,10 +3,12 @@
     <img src="/assets/images/chucksPickThreeLogo.png" alt="logo" width="50px" height="50px">
   </div>
   <div class="header-img-container">
-    <img src="/assets/images/ChucksPick3HeaderDesktop.png" class="desktopHeader" alt="logo" width="700px"
+    <img src="src/app/assets/images/ChucksPick3HeaderDesktop.png" class="desktopHeader" alt="logo" width="700px"
```
Comment (pouncilt) [inline chucks-pick3-web/angular-app/src/app/shared/components/app-header.component/app-header.component.html]: Not Sure about this change.  Does this still work?  I think it is okay to start paths to images and css files with just `assets/images` and `assets/css`;  Please revert back.

File: chucks-pick3-web/angular-app/src/app/shared/components/app-header.component/app-header.component.html [hunk 17]
```diff
@@ -3,10 +3,12 @@
...
+         height="100px">
+    <img src="src/app/assets/images/ChucksPick3HeaderTablet.png" class="tabletHeader" alt="logo" width="1000px"
```
Comment (pouncilt) [inline chucks-pick3-web/angular-app/src/app/shared/components/app-header.component/app-header.component.html]: Not Sure about this change.  Does this still work?  I think it is okay to start paths to images and css files with just `assets/images` and `assets/css`;  Please revert back.

File: chucks-pick3-web/angular-app/src/app/shared/components/app-header.component/app-header.component.html [hunk 18]
```diff
@@ -3,10 +3,12 @@
...
+         height="100px">
+    <img src="src/app/assets/images/ChucksPick3HeaderMobile.png" class="mobileHeader" alt="logo" width="1000px"
```
Comment (pouncilt) [inline chucks-pick3-web/angular-app/src/app/shared/components/app-header.component/app-header.component.html]: Not Sure about this change.  Does this still work?  I think it is okay to start paths to images and css files with just `assets/images` and `assets/css`;  Please revert back.

File: chucks-pick3-web/angular-app/src/app/shared/components/app.component/app.component.html [hunk 19]
```diff
@@ -5,13 +5,13 @@
...
-  <div class= "splash-container" *ngIf="showSplashPage === true">
-    <div class="splash-logo-container">
-      <img src="assets/images/chucksPickThreeLogo.png" alt="logo" width="300px" height="300px"/>
-    </div>
-    <div class=" link-container">
-      <span><a [routerLink]="'/home'" (click)="showSplashView(false)">LEARN MORE</a></span>
-    </div>
+<div class="splash-container" *ngIf="showSplashPage === true">
+  <div class="splash-logo-container">
+    <img src="src/app/assets/images/chucksPickThreeLogo.png" alt="logo" width="300px" height="300px"/>
```
Comment (pouncilt) [inline chucks-pick3-web/angular-app/src/app/shared/components/app.component/app.component.html]: Not Sure about this change.  Does this still work?  I think it is okay to start paths to images and css files with just `assets/images` and `assets/css`;  Please revert back.

File: chucks-pick3-web/angular-app/src/app/components/thank-you.component/thank-you.component.ts [hunk 20]
```diff
@@ -0,0 +1,16 @@
+import {Component, OnInit} from '@angular/core';
+
+@Component({
+  selector: 'app-thank-you',
+  templateUrl: './thank-you.component.html',
+  styleUrls: ['./thank-you.component.css']
+})
+export class ThankYouComponent implements OnInit {
```
Comment (pouncilt) [inline chucks-pick3-web/angular-app/src/app/components/thank-you.component/thank-you.component.ts]: Lets rename this component to SignUpConfirmationComponent.  The directory and file name should be: 

1. `src/app/components/sign-up-confirmation.component/sign-up-confirmation.component.ts`
2. `src/app/components/sign-up-confirmation.component/sign-up-confirmation.component.html`
3. `src/app/components/sign-up-confirmation.component/sign-up-confirmation.component.css`

File: chucks-pick3-web/angular-app/src/app/components/info-generate.component/info-generate.component.html (hunk 10 above)
Comment (ocampd1) [reply chucks-pick3-web/angular-app/src/app/components/info-generate.component/info-generate.component.html]: reverted back to assets/images.

File: chucks-pick3-web/angular-app/src/app/components/info-interface.component/info-interface.component.html (hunk 11 above)
Comment (ocampd1) [reply chucks-pick3-web/angular-app/src/app/components/info-interface.component/info-interface.component.html]: reverted back to assets/images.

File: chucks-pick3-web/angular-app/src/app/components/info-generate.component/info-generate.component.html (hunk 10 above)
Comment (ocampd1) [reply chucks-pick3-web/angular-app/src/app/components/info-generate.component/info-generate.component.html]: reverted back to assets/images.

File: chucks-pick3-web/angular-app/src/app/components/info-requirements.component/info-requirements.component.html (hunk 13 above)
Comment (ocampd1) [reply chucks-pick3-web/angular-app/src/app/components/info-requirements.component/info-requirements.component.html]: reverted back to assets/images.

File: chucks-pick3-web/angular-app/src/app/components/info.component/info.component.html (hunk 15 above)
Comment (ocampd1) [reply chucks-pick3-web/angular-app/src/app/components/info.component/info.component.html]: reverted back to assets/images.

File: chucks-pick3-web/angular-app/src/app/shared/components/app-header.component/app-header.component.html (hunk 16 above)
Comment (ocampd1) [reply chucks-pick3-web/angular-app/src/app/shared/components/app-header.component/app-header.component.html]: reverted back to assets/images.

File: chucks-pick3-web/angular-app/src/app/shared/components/app-header.component/app-header.component.html (hunk 18 above)
Comment (ocampd1) [reply chucks-pick3-web/angular-app/src/app/shared/components/app-header.component/app-header.component.html]: reverted back to assets/images.

File: chucks-pick3-web/angular-app/src/app/components/thank-you.component/thank-you.component.ts (hunk 20 above)
Comment (ocampd1) [reply chucks-pick3-web/angular-app/src/app/components/thank-you.component/thank-you.component.ts]: Completed changes for naming convention of sign up component.

File: chucks-pick3-web/angular-app/src/app/shared/components/app.component/app.component.html (hunk 19 above)
Comment (ocampd1) [reply chucks-pick3-web/angular-app/src/app/shared/components/app.component/app.component.html]: reverted to assets/images

File: chucks-pick3-web/angular-app/src/app/shared/components/app-header.component/app-header.component.html (hunk 17 above)
Comment (ocampd1) [reply chucks-pick3-web/angular-app/src/app/shared/components/app-header.component/app-header.component.html]: reverted to assets/images

File: chucks-pick3-web/angular-app/src/app/components/info-view.component/info-view.component.html (hunk 14 above)
Comment (ocampd1) [reply chucks-pick3-web/angular-app/src/app/components/info-view.component/info-view.component.html]: reverted to assets/images

File: chucks-pick3-web/angular-app/src/app/components/info-interface.component/info-interface.component.html (hunk 12 above)
Comment (ocampd1) [reply chucks-pick3-web/angular-app/src/app/components/info-interface.component/info-interface.component.html]: reverted to assets/images

File: chucks-pick3-web/angular-app/src/app/components/info-generate.component/info-generate.component.html (hunk 9 above)
Comment (ocampd1) [reply chucks-pick3-web/angular-app/src/app/components/info-generate.component/info-generate.component.html]: reverted to assets/images

File: chucks-pick3-web/angular-app/src/app/components/home.component/home.component.html (hunk 8 above)
Comment (ocampd1) [reply chucks-pick3-web/angular-app/src/app/components/home.component/home.component.html]: reverted to assets/images

File: chucks-pick3-web/angular-app/src/app/components/home.component/home.component.html (hunk 7 above)
Comment (ocampd1) [reply chucks-pick3-web/angular-app/src/app/components/home.component/home.component.html]: reverted to assets/images.

File: chucks-pick3-web/angular-app/src/app/components/home.component/home.component.html (hunk 6 above)
Comment (ocampd1) [reply chucks-pick3-web/angular-app/src/app/components/home.component/home.component.html]: reverted to assets/images
